connect/json/src/main/java/org/apache/kafka/connect/json/JsonConverter.java

```diff
@@ -143,7 +143,7 @@ public class JsonConverter implements Converter, HeaderConverter, Versioned {
     // names specified in the field
     private static final HashMap<String, LogicalTypeConverter> LOGICAL_CONVERTERS = new HashMap<>();
 
-    private static final JsonNodeFactory JSON_NODE_FACTORY = JsonNodeFactory.withExactBigDecimals(true);
+    private static final JsonNodeFactory JSON_NODE_FACTORY = new JsonNodeFactory(true);
```

Comment:
Can you clarify why we're making this changes?

Reply:
@mimaison  Sorry I missed answering this point, the change is made because the static method is deprecated 

```
    @Deprecated
    public static JsonNodeFactory withExactBigDecimals(boolean bigDecimalExact) {
        return new JsonNodeFactory(bigDecimalExact);
    }
 
```
and the best available replacement for it is the the constructor 

```
    public JsonNodeFactory(boolean bigDecimalExact) {
        this._cfgBigDecimalExact = bigDecimalExact;
    }
```